core/src/main/java/org/elasticsearch/common/settings/Settings.java

```diff
@@ -621,6 +619,112 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         }
         return builder;
     }
+    public static Settings.Builder fromXContent(XContentParser parser, Settings.Builder builder) throws IOException {
+        return fromXContent(parser, builder, true, false);
+    }
+
+    private static Settings.Builder fromXContent(XContentParser parser, Settings.Builder builder, boolean allowNullValues, boolean
+        validateEndOfStream)
```

Comment:
Odd place to break the line, can at least the type of this var be moved down with it?